qa/unittests/requirements.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/ESA-APEx/esa-apex-toolbox-python.git@main
 pytest>=8.2.0
 moto[s3, server]>=5.0.13
 dirty-equals>=0.8.0
+fusets>=2.0.1
```

Comment:
No, it should not be added here.
fusets is not a dependency for the unit tests.

I meant to add a requirements.txt file to the openeo_udp/fusets_mogpr/folder